templates/ts/custom-copilot-rag-azure-ai-search/env/.env.dev.user.tpl

```diff
@@ -0,0 +1,50 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
+
+# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
+SECRET_BOT_PASSWORD=
+{{#useOpenAI}}
+{{#openAIKey}}
+SECRET_OPENAI_API_KEY='{{{openAIKey}}}'
+{{/openAIKey}}
+{{^openAIKey}}
+SECRET_OPENAI_API_KEY=
+{{/openAIKey}}
+{{/useOpenAI}}
+{{#useAzureOpenAI}}
+{{#azureOpenAIKey}}
+SECRET_AZURE_OPENAI_API_KEY='{{{azureOpenAIKey}}}'
+{{/azureOpenAIKey}}
+{{^azureOpenAIKey}}
+SECRET_AZURE_OPENAI_API_KEY=
+{{/azureOpenAIKey}}
+{{#azureOpenAIEndpoint}}
+AZURE_OPENAI_ENDPOINT='{{{azureOpenAIEndpoint}}}'
+{{/azureOpenAIEndpoint}}
+{{^azureOpenAIEndpoint}}
+AZURE_OPENAI_ENDPOINT=
+{{/azureOpenAIEndpoint}}
+{{#azureOpenAIDeploymentName}}
+AZURE_OPENAI_DEPLOYMENT_NAME='{{{azureOpenAIDeploymentName}}}'
```

Comment:
this value and `azureOpenAIEmbeddingDeploymentName` wont be questioned when scaffolding. so there will be a bug. 
We only need to give a blank value for this key here, and mention in Readme file to remind users to fill these two values.
Same in all .env.*.user files